File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/22/22.go
// Q22: Print diamond star pattern.
// Input: An integer n
// Output: Diamond shape with stars

package main

import (
    "fmt"
    "strings"
)

func main() {
    var n int
    fmt.Scan(&n)
    for i := 1; i <= n; i++ {
        fmt.Println(strings.Repeat(" ", n-i) + strings.Repeat("*", 2*i-1))
    }
    for i := n - 1; i >= 1; i-- {
        fmt.Println(strings.Repeat(" ", n-i) + strings.Repeat("*", 2*i-1))
    }
}

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/22/22.js
// Q22: Print diamond star pattern.
// Input: An integer n
// Output: Diamond shape with stars

const readline = require('readline');
const rl = readline.createInterface({ input: process.stdin });
rl.on('line', (line) => {
    const n = parseInt(line);
    for (let i = 1; i <= n; i++) {
        console.log(" ".repeat(n - i) + "*".repeat(2 * i - 1));
    }
    for (let i = n - 1; i >= 1; i--) {
        console.log(" ".repeat(n - i) + "*".repeat(2 * i - 1));
    }
    rl.close();
});
